boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-ipc_conf.dtsi

```diff
@@ -41,6 +41,13 @@
 				 <&cpuppr_vevif 12>;
 		};
 
+		cpuapp_cpuflpr_ipc: ipc-2-14 {
+			compatible = "zephyr,ipc-icmsg";
+			status = "disabled";
+			mboxes = <&cpuapp_bellboard 14>,
+				 <&cpuflpr_vevif 13>;
```

Comment:
This one should be "12" to be consistent with the design in https://github.com/nrfconnect/doc-internal/pull/112

Reply:
Note: I see that for cpuflpr vevif, the tasks mask is 0xffff0000, so if I got this correctly, only 31-16 can be used (driver is designed to handle this way right now).

Reply:
Do you mean VEVIF task 12 does not work? If that's the case, the channels allocation should be redesigned and doc-internal updated to reflect this

Reply:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/25011557/c0759624-ac41-4974-9740-aec11612abc4)

It doesn't seem to work (just tested quickly)

Reply:
Could you try `<&cpuflpr_vevif 16>;`? If it works as expected I'll update the docs accordingly.

Reply:
yes, 16 works fine